makedoc.g

```diff
@@ -15,7 +15,18 @@ LoadPackage("AutoDoc");
 AutoDoc(rec( 
     autodoc := true,
     scaffold := rec(
-        includes := [ "Tutorials.xml", "Comments.xml" ]
+        includes := [ "Tutorials.xml", 
+                      "Comments.xml" ],
+        bib := "bib.xml", 
+        gapdoc_latex_options := rec( EarlyExtraPreamble := """
+            \usepackage[all]{xy} 
+            \newcommand{\bbZ} {\mathbb{Z}}
+        """ ),  
+        entities := rec( 
+            VERSION := "2017.09.15", 
```

Comment:
Huh? Is this supposed to be the current version? Then I strongly object, as we are *guaranteed* to forget to update it with a release (though I guess that could be fixed by changing it to `VERSION := ~.Version`).

Or is it supposed to be "some version number, possibly old, which people should set as minimal required AutoDoc version? In that case, the name `VERSION` isn't so great, and there should also be a comment here explaining that use.

Reply:
. . . and therefore removed entity VERSION.